gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePublishPlugin.java

```diff
@@ -39,8 +39,8 @@ public void apply(Project project) {
         // Configure publishing
         project.getExtensions().configure(PublishingExtension.class, publishing -> {
             publishing.publications(publications -> {
-                publications.create("conjure", MavenPublication.class, mavenPublication ->
-                        mavenPublication.artifact(compileIr.getOutputFile(), mavenArtifact -> {
+                publications.create("conjure", MavenPublication.class, mavenPublication -> mavenPublication.artifact(
```

Comment:
I think this is a regression as it looks worse, PR to fix it up: https://github.com/palantir/palantir-java-format/pull/169